etc/sonar.sh

```diff
@@ -17,15 +17,24 @@ if [ $(curl -I -s https://sonar-lightblue.rhcloud.com/ -s | head -n1 | grep "HTT
 fi
 
 
-# kill any previous port forward
+# kill any previous port forward and the previous mark file  about its status
 pkill -9 -f "rhc port-forward sonar [-]n lightblue"
+rm -f rhc-port-forward-status
 
-# forward ports from openshift (assumes current server is openshift online and you're a member of the lightblue namespace)
-rhc port-forward sonar -n lightblue &
+# forward ports from openshift (assumes current server is openshift online and you're a member of the lightblue namespace). A new mark file will eb created if the return value of the rhc command is different than 0
+rhc port-forward sonar -n lightblue || echo $? > rhc-port-forward-status  &
 
 # wait a bit for port forwarding to fire up
 sleep 10
 
+# if the file exists, log the error and exit 1
+if [ -f "rhc-port-forward-status" ]
```

Comment:
I don't think we should do that as the '&' would make the process run asynchronously in background , which it can take some time even to connect to the openshift, so the rhc command could not make the result in time to the if statement gives the right/expected output